Dockerfile

```
@@ -0,0 +1,45 @@
+FROM mcr.microsoft.com/playwright:v1.58.1-jammy
+
+ENV DEBIAN_FRONTEND=noninteractive \
+    NODE_ENV=development \
+    PLAYWRIGHT_SKIP_BROWSER_DOWNLOAD=0 \
+    PLAYWRIGHT_BROWSERS_PATH=/ms-playwright \
+    PATH=/usr/local/go/bin:$PATH
+
+# Optional: Go toolchain for the dashboard TUI (./dashboard).
+# Small footprint, keeps full feature parity with the README setup.
+ARG GO_VERSION=1.23.4
+RUN set -eux; \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)